modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -2824,17 +2823,23 @@
                                       "example": "USA"
                                     },
                                     "zipFirstFive": {
-                                      "description": "Zip code (First 5 digits) for the Veteran's current mailing address.",
+                                      "description": "Zip code (First 5 digits) for the Veteran's current mailing address. Required if 'country' is 'USA'.",
                                       "type": "string",
                                       "pattern": "^\\d{5}?$",
-                                      "example": "12345"
+                                      "example": "12345",
+                                      "nullable": true
                                     },
                                     "zipLastFour": {
                                       "description": "Zip code (Last 4 digits) for the Veteran's current mailing address.",
                                       "type": "string",
                                       "pattern": "^\\d{4}?$",
                                       "example": "6789",
                                       "nullable": true
+                                    },
+                                    "internationalPostalCode": {
```

Comment:
should we add a description to this specifically stating `if not 'USA'` just to clarify?